app_legacy/Helpers/render/layout.php

```diff
@@ -105,16 +107,28 @@ function RenderToolbar(): void
     echo "</ul>";
 
     echo "<ul>";
+    echo "<li class='dropdown-header'>Miscellaneous</li>";
     echo "<li><a href='/gameList.php'>All Games</a></li>";
     // echo "<li><a href='/popularGames.php'>Most Played</a></li>";
     echo "<li><a href='/setRequestList.php'>Most Requested</a></li>";
+    echo "<li class='dropdown-header'>Hubs</li>";
+    echo "<li><a href='/gameList.php?s=6&c=100&f=1'>Hub List</a></li>";
+    echo "<li><a href='/game/6914'>Central Hub</a></li>";
+    echo "<li><a href='/game/9553'>Genre & Subgenre Hub</a></li>";
+    echo "<li><a href='/game/5771'>Series Hub</a></li>";
+    echo "<li><a href='/game/3105'>Community Events</a></li>";
+    echo "<li><a href='/game/3273'>Developer Events</a></li>";
     echo "</ul>";
     echo "</div>";
     echo "</li>";
 
     echo "<li><a href='#'>Achievements</a>";
     echo "<div>";
     echo "<ul>";
+    echo "<li><a href='/claimlist.php?s=9&f=8109'>New Sets & Revisions</a></li>";
+    echo "<li><a href='/claimlist.php'>Sets in Progress</a></li>";
+    echo "<li><a href='/setRequestList.php'>Most Requested Sets</a></li>";
```

Comment:
This is redundant with the `Most Requested` link in the Games menu. Should `New Sets` and `Sets in Progress` be moved there?

Reply:
Got rid of `Most Requested` in the Achievements menu. I guess I overlooked that one. Moved the others to the Games menu since it makes sense.